IndomitableIbises/Bangazon-Sprint-2
# Computer model ea

## Link to Ticket
References # [Link](https://github.com/IndomitableIbises/Bangazon-Sprint-2/issues/8)

## Description of Proposed Changes

- This pull request adds the Computer model and the ComputerListView.  Upon migrating the database, you should be able to open db browser and view the computer table.

## Steps to Test
git fetch --all
git checkout computerModel-ea
rm -rf migrations folder
rm sqlite database
makemigration
migrate
open dbbrowser and view the computer table
should see the following fields agreed upon in ERD.



## Impacted Areas in Application

List general components of the application that this PR will affect:

*

## Mentions @username

Tag users that need to review this code

## Definition of Done

1. The project must be fully documented. This includes the following:
   1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
   1. Every class must be documented with purpose, author, and methods.
   1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each
